service/sync_gateway_service_install.sh

```diff
@@ -47,7 +47,7 @@ usage() {
 }
 
 ostype() {
-  lsb_release 2>&1 > /dev/null
+  _lsb_release_exists=$(command -v lsb_release)
   if [ $? -eq 0 ]; then
```

Comment:
I think this is as portable and a more common way of writing this, with the benefit of also catching the odd case of `lsb_release` not being executable

```suggestion
  if [ -x "$(command -v lsb_release)" ]; then
```